### ISSUE TITLE
Contrast for .wdn-button-complement

### ISSUE THREAD

**Issue by mfairchild365** [unl/wdntemplates#702]
WCAG2AA
Discovered with: http://squizlabs.github.io/HTML_CodeSniffer/

```
This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 4.5:1. Recommendation: change background to #00892c.
```

Code Snippet:

```
<a href="#" class="wdn-button wdn-button-complement">This is my button</a>
```
